bin/sozo/src/commands/dev.rs

```diff
@@ -243,7 +243,7 @@ impl DevArgs {
                 world_address = address;
             }
             Err(error) => {
-                log::error!("Error: {error:?}");
+                log::error!(target: "sozo::cli::commands::dev", "Error", error = ?error);
```

Comment:
```suggestion
                log::error!(target: "sozo::cli::commands::dev", "Migrating world.", error = ?error, address = ?address);
```

Ideally we provide a bit more error context here